src/overlays/actors/ovl_Obj_Spidertent/z_obj_spidertent.c

```diff
@@ -28,118 +35,789 @@ const ActorInit Obj_Spidertent_InitVars = {
+        {
+            ELEMTYPE_UNK4,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00000800, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
+        { { { 17.799999237060547f, 0.0f, -87.19999694824219f },
+            { -83.4000015258789f, 0.0f, -19.700000762939453f },
+            { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
     },
     {
-        { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0x00000800, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
-        { { { -60.900001525878906f, 0.0f, 59.099998474121094f }, { 17.799999237060547f, 0.0f, 81.5999984741211f }, { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
+        {
+            ELEMTYPE_UNK4,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00000800, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
+        { { { 57.20000076293945f, 0.0f, -47.79999923706055f },
+            { 17.799999237060547f, 0.0f, -87.19999694824219f },
+            { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
     },
     {
-        { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0x00000800, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
-        { { { -83.4000015258789f, 0.0f, -19.700000762939453f }, { -60.900001525878906f, 0.0f, 59.099998474121094f }, { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
+        {
+            ELEMTYPE_UNK4,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00000800, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
+        { { { 51.599998474121094f, 0.0f, 14.100000381469727f },
+            { 57.20000076293945f, 0.0f, -47.79999923706055f },
+            { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
     },
+};
+
+static ColliderTrisInit sTrisInit2 = {
     {
-        { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0x00000800, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
-        { { { 17.799999237060547f, 0.0f, -87.19999694824219f }, { -83.4000015258789f, 0.0f, -19.700000762939453f }, { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_TRIS,
     },
+    6,
+    sTrisElementsInit2,
+};
+
+typedef struct {
+    /* 0x00 */ UNK_PTR unk_00;
+    /* 0x04 */ CollisionHeader* unk_04;
+    /* 0x08 */ ColliderTrisInit* unk_08;
+    /* 0x0C */ f32 unk_0C;
+    /* 0x10 */ f32 unk_10;
+    /* 0x14 */ f32 unk_14;
+    /* 0x18 */ f32 unk_18;
+    /* 0x1C */ f32 unk_1C;
+    /* 0x20 */ f32 unk_20;
+    /* 0x24 */ f32 unk_24;
+} _struct_D_80B31350_0x28;
+
+_struct_D_80B31350_0x28 D_80B31350[] = {
```

Comment:
```suggestion
} ObjSpidertent_struct_80B31350;

ObjSpidertent_struct_80B31350 D_80B31350[] = {
```
I'm not sure what went wrong with this name but it can be simplified quite a bit.

Reply:
This is how m2c outputs them, I just didn't rename them because I never give them proper names anyway. This seems just as good as my usual names lol.